package.json

```diff
@@ -134,6 +136,7 @@
     "http-node": "^1.2.0",
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
+    "query-string": "^6.1.0",
     "react-chrome-redux": "^1.5.1",
     "throttleit": "^1.0.0",
     "webtorrent": "github:brave/webtorrent#600664249258dd7e893789e991d4cc5f481a7bae"
```

Comment:
general question, is there a plan for keeping these github forks up to date? we have had some security issues with webtorrent in the past, and i think by forking them in this way, we will no longer get security reports for them with `npm audit`.

Reply:
https://github.com/brave/brave-browser/issues/856, in the long-term, we want to solve the browser field issue without forking them.
In the short term, I could keep an eye from webtorrent upstream changes and rebase onto their latest releases, I'm not sure if that's sufficient tho.
I wonder if we have guidelines on how to follow upstream changes for projects we're forking, like how frequently should we check or update to the upstream?

Reply:
rebasing them sgtm for now. we don't really have guidelines right now; probably once per release? if there is a release checklist, this should be added to it.

Reply:
Per release sounds good to me.
I didn't see one in our wiki tho I might missed it, @bbondy, please correct me if there is.
I could create bugs for next few releases first and mark with milestone labels, how does that sound, @diracdeltas ?